lib/openstudio-standards/standards/necb/ECMS/nv.rb

```diff
@@ -0,0 +1,187 @@
+              zone_htg_thermostat_sch_name = zone_htg_thermostat_sch.name
+              zone_htg_sp_schedule = zone_htg_thermostat_sch.to_ScheduleRuleset.get
+              puts zone_htg_sp_schedule.name
+              min_Tin_schedule = zone_htg_sp_schedule.clone(model).to_ScheduleRuleset.get
+              min_Tin_schedule.setName('natural_ventilation_min_Tin_schedule')
+              ## default days/weekdays
+              min_Tin_schedule_defaultDay = min_Tin_schedule.defaultDaySchedule
+              min_Tin_schedule_defaultDay.setName('natural_ventilation_min_Tin_schedule_defaultDay')
+              min_Tin_schedule_defaultDay_times = min_Tin_schedule_defaultDay.times
+              min_Tin_schedule_defaultDay_values = min_Tin_schedule_defaultDay.values
+              min_Tin_schedule_defaultDay_values_adjusted = min_Tin_schedule_defaultDay_values.map { |i| i - setpoint_adjustment_for_nv }
+              i = 0.0
+              min_Tin_schedule_defaultDay_times.each do |time|
+                min_Tin_schedule_defaultDay.addValue(time, min_Tin_schedule_defaultDay_values_adjusted[i])
+                i += 1.0
+              end
+
+
+            end
+          end
+        end
+
+        ##### Calculate how many windows a space has.
+        # The total number of windows is used to divide OA/person and OA/FloorArea of the space by it (i.e. number of windows).
+        # In this way, NV-driven OA in each space would be avoided to be more than required.
+        space.surfaces.sort.each do |surface|
+          surface.subSurfaces.sort.each do |subsurface|
+            # puts subsurface.name.to_s
+            if (subsurface.subSurfaceType == 'OperableWindow' || subsurface.subSurfaceType == 'FixedWindow') && subsurface.outsideBoundaryCondition == 'Outdoors'  #TODO: Question: Should I change window type to only 'OperableWindow'? For testing purposes, I have included the window type of 'FixedWindow' as well.
+              number_of_windows += 1.0
+            end
+          end
+        end
+        # puts "#{space.name.to_s} has #{number_of_windows} window(s)"
+        oa_per_person_normalized_by_number_of_windows = outdoor_air_flow_per_person/number_of_windows
+        oa_per_floor_area_normalized_by_number_of_windows = outdoor_air_flow_per_floor_area/number_of_windows
+
+        ##### Add NV in each space that has window(s) using two objects: "ZoneVentilation:DesignFlowRate" and "ZoneVentilation:WindandStackOpenArea"
+        space.surfaces.sort.each do |surface|
+          surface.subSurfaces.sort.each do |subsurface|
+            # puts subsurface.name.to_s
+            if (subsurface.subSurfaceType == 'OperableWindow' || subsurface.subSurfaceType == 'FixedWindow') && subsurface.outsideBoundaryCondition == 'Outdoors'  #TODO: Question: Should I change window type to only 'OperableWindow'? For testing purposes, I have included the window type of 'FixedWindow' as well.
```

Comment:
To answer your question. Add to both as you did. You can remove the comment.

Reply:
Removed the comment.